source/administration/mmctl-cli-tool.rst

```diff
@@ -0,0 +1,2318 @@
+
+**Description**
+
+  List all active and inactive plugins installed on your Mattermost server.
+
+**Format**
+
+.. code-block:: sh
+
+    mmctl plugin list [flags]
+
+**Examples**
+
+.. code-block:: sh
+
+    plugin list
+
+**Options**
+
+.. code-block:: sh
+
+   -h, --help   help for list
+
+**Options Inherited from Parent Commands**
+
+.. code-block:: sh
+
+  --format string   the format of the command output [plain, json] (default "plain")
+
+
+mmctl post
+------------
+
+Management of posts.
+
+Child Commands
+  -  `mmctl post create`_ - Create a post
+  -  `mmctl post list`_ - List posts
+
+**Options**
+
+.. code-block:: sh
+
+   -h, --help   help for post
+
+mmctl post create
+^^^^^^^^^^^^^^^^^^
+
+**Description**
+
+  Create a post.
+
+**Format**
+
+.. code-block:: sh
+
+    mmctl post create [flags]
+
+**Examples**
+
+.. code-block:: sh
+
+    post create myteam:mychannel --message "some text for the post"
+
+**Options**
+
+.. code-block:: sh
+
+  -h, --help              help for create
+  -m, --message string    Message for the post
+  -r, --reply-to string   Post id to reply to
+
+**Options Inherited from Parent Commands**
+
+.. code-block:: sh
+
+    --format string   the format of the command output [plain, json] (default "plain")
+
+mmctl post list
+^^^^^^^^^^^^^^^^
+
+**Description**
+
+  List posts for a channel.
+
+**Format**
+
+.. code-block:: sh
+
+   mmctl post list [flags]
+
+**Examples**
+
+.. code-block:: sh
+
+    post list myteam:mychannel
+    post list myteam:mychannel --number 20
+
+**Options**
+
+.. code-block:: sh
+
+  -f, --follow       Output appended data as new messages are posted to the channel
+  -h, --help         help for list
+  -n, --number int   Number of messages to list (default 20)
+  -i, --show-ids     Show posts ids
+
+**Options Inherited from Parent Commands**
+
+.. code-block:: sh
+
+    --format string   the format of the command output [plain, json] (default "plain")
+
+mmctl roles
+-----------
+
+mmctl team
+----------
+
+Management of teams.
+
+Child Commands
+  -  `mmctl team add`_ - Add teams
+  -  `mmctl team archive`_ - Archive teams
+  -  `mmctl team create`_ - Create teams
+  -  `mmctl team delete`_ - Delete teams
+  -  `mmctl team list`_ - List teams
+  -  `mmctl team remove`_ - Remove teams
+  -  `mmctl team rename`_ - Rename teams
+  -  `mmctl team search`_ - Search teams
+
+**Options**
+
+.. code-block:: sh
+
+  -h, --help   help for team
+
+mmctl team add
+^^^^^^^^^^^^^^^^
+
+**Description**
+
+  Add specified users to a team.
+
+**Format**
+
+.. code-block:: sh
+
+    mmctl team add [team] [users] [flags]
+
+**Examples**
+
+.. code-block:: sh
+
+    team add myteam user@example.com username
+
+**Options**
+
+.. code-block:: sh
+
+   -h, --help   help for add
+
+**Options Inherited from Parent Commands**
+
+.. code-block:: sh
+
+    --format string   the format of the command output [plain, json] (default "plain")
+
+mmctl team archive
+^^^^^^^^^^^^^^^^
+
+*In progress*
```

Comment:
In progress

Reply:
Same as previous comment - I worked off the migration sheet and the repo and thought it might be helpful to list them all but mark which ones are still being worked on. Can definitely remove this.

Reply:
Removed.